src/summary/history/notes-card.component.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { match } from "react-router";
+import SummaryCard from "../cards/summary-card.component";
+import SummaryCardRow from "../cards/summary-card-row.component";
+import SummaryCardRowContent from "../cards/summary-card-row-content.component";
+import HorizontalLabelValue from "../cards/horizontal-label-value.component";
+import { encounterResource } from "./encounter.resource";
+import VerticalLabelValue from "../cards/vertical-label-value.component";
+import dayjs from "dayjs";
+
+export default function NotesCard(props: NotesCardProps) {
+  const [patientNotes, setPatientNotes] = React.useState(null);
+
+  React.useEffect(() => {
+    const abortController = new AbortController();
+
+    encounterResource(props.currentPatient.identifier[0].value, abortController)
+      .then(notes => setPatientNotes(notes))
+      .catch(error => {
+        setTimeout(() => {

```

Comment:
checkout how to use `createErrorHandler()` from `@openmrs/esm-error-handling`
```suggestion
        createErrorHandler();
```